### ISSUE TITLE
UserWarning: Input dict contained keys [...] which did not match any model input. They will be ignored by the model.

### ISSUE THREAD

**Issue by azagniotov** [tensorflow/ranking#269]
I am am using the following example to train a model using a toy data (I can provide ELWC structure if needed): https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/examples/keras/keras_m2e_tfrecord.py 

on the following TF stack:
- tensorflow==2.5.0
- tensorflow-serving-api==2.5.1 
- tensorflow-ranking==0.4.0

During training I see the following warning about a missing key `stars`and and I do not know if this is a warning I should worry about. The `stars` is the name of my label feature in ELWC

```
INFO:tensorflow:Calling model_fn.
```
```
/home/ec2-user/anaconda3/envs/python3/lib/python3.6/site-packages/tensorflow/python/keras/engine/functional.py:591: UserWarning: Input dict contained keys ['stars'] which did not match any model input. They will be ignored by the model.
  [n for n in tensors.keys() if n not in ref_input_names])
```
```
INFO:tensorflow:Done calling model_fn.
```

Could you please advise?

**Comment by xuanhuiwang** [comment]
This doesn't affect the training. You can `pop` the feature `stars` from the input features when constructing the labels.

**Comment by azagniotov** [comment]
@xuanhuiwang thank you. Right, but the label is already being `pop`ed in the following code as per the aforementioned example:
```
features = tf.compat.v1.data.make_one_shot_iterator(dataset).get_next()
label = tf.squeeze(features.pop(_LABEL_FEATURE), axis=2)
label = tf.cast(label, tf.float32)
```
but, thank you again for confirming about the training,
